pg_es_fdw/__init__.py

```diff
@@ -227,14 +245,18 @@ def _convert_response_row(self, row_data, columns, query):
                     or column == self.score_column
                 ]
                 + [(self.query_column, query)]
+                + [(self.sort_column, sort)]
             )
-        return {
-            column: self._convert_response_column(column, row_data)
-            for column in columns
-            if column in row_data["_source"]
-            or column == self.rowid_column
-            or column == self.score_column
-        }
+        return dict(
+            [
+                (column, self._convert_response_column(column, row_data))
+                for column in columns
+                if column in row_data["_source"]
+                or column == self.rowid_column
+                or column == self.score_column
+            ]
+            + [(self.sort_column, sort)]
+        )
```

Comment:
It's easier to read this if you assign the original dict comprehension to a variable and then assign the sort_column key.